### DISCUSSION TITLE
[OnPrem] GlideRecord script does not change

### DISCUSSION THREAD

**Discussion by mikeymuis** [arnoudkooi/SN-Utils#490] [General]
Hi,

When I create a query/filter on an OnPrem instance (doesn't matter which table) and then open the GlideRecord tab via SNUtils nothing changes on "(bakcround)script".
No matter what the filter looks like, it always remains the same:

```
var grNull = new GlideRecord('null');
if (grNull.get('')) {
    gs.info(': ' + grNull.getValue(''));
}
```

On Premis Instance
Build name: Tokyo
Build date: 03-24-2023_1357
Build tag: glide-tokyo-07-08-2022__patch8-03-16-2023
UI: Classic
SN Utils 7.3.3.0 (earlier versions have the same problem).
Chrome Version 121.0.6167.140 (Official Build) (64-bit)

**Comment by arnoudkooi** [comment]
Did you give permissions to your specific URL in the extension settings page?
![Google Chrome20240408 - 1406](https://github.com/arnoudkooi/ServiceNow-Utils/assets/840781/90be3947-3142-4be8-9fce-9f4eab1df15d)

**Reply by mikeymuis** [reply]
I have "On all sites" selected, just tried "On specific sites" but same result.

**Comment by arnoudkooi** [comment]
Can you add a screenshot of the page and see if there are console errors.

**Comment by mikeymuis** [comment]
![error](https://github.com/arnoudkooi/ServiceNow-Utils/assets/102667263/9b7ef5cf-7f6e-4ed8-a349-a6100cadee17)

Is this you are looking for?

**Comment by arnoudkooi** [comment]
Is that list v3? That is no longer supported. If you do use it, I would recommend disabling it

**Reply by mikeymuis** [reply]
Thats it! Thanks!
